project-progression-policy.md

```diff
@@ -20,6 +20,14 @@ Some ways a project can benefit by becoming a Consortium-recognized project incl
 4. Project Support: The Consortium can provide limited funds to support project communication and infrastructure costs for CI/CD.
 5. Outreach: Part of the Consortium's mission is to promote the use of confidential computing with various communities and hence its recognized projects, to deliver that outcome.
 6. Internships: The [TAC has committed](https://lists.confidentialcomputing.io/g/main/files/TAC/Meetings/2021/09-Sept/TAC%20Minutes%202021-09-09.pdf) to fund one intern per CCC project that commits to the list of "Mentor Responsibilities" on the [Outreachy mentor page](https://www.outreachy.org/mentor/).
+7. Test infrastructure: Up to $7500 in budget for hardware and software per year.
+8. Collaboration: Interns are provided with [Collaboration tools](mailto:operations@confidentialcomputing.io).
```

Comment:
In the slide referenced from the issue there are 4 tools listed. 3 are already in this document. Just need to list zoom.
```suggestion
8. Video conferencing support: A zoom account for community meetings.
```